### ISSUE TITLE
FR: Enhancement: Streamlining FX naming conventions for {{target.fx.name}} 

### ISSUE THREAD

**Issue by PucksEvilTwin** [helgoboss/realearn#772]
This placeholder could be improved quite a bit with some regex before it reaches the control surface.  With minimal available characters on displays, it would be a good idea to make the following wholesale changes:

1.  Cut out any FX type prefix.  VST, VST3, JS etc. so that VST3: Lindell SBC = Lindell SBC (so anything before ": ")
2.  Cut out anything after "(" these are normally just manufacturers names.

**Comment by helgoboss** [comment]
Makes sense. However, I wouldn't change the content of `target.fx.name` itself - it just reflects whatever name the FX instance has, so it can be also a custom user-defined name. Instead, I would introduce new placeholders that reflect the *original* plug-in name. First draft:

- `target.fx.type.full_name` ("VSTi: ReaSynth (Cockos)")
- `target.fx.type.name` ("ReaSynth")
- `target.fx.type.manufacturer` ("Cockos")
- `target.fx.type.type` ("VST")
- `target.fx.type.sub_type` ("VSTi")
- `target.fx.type.file_name` ("reasynth.vst.dylib")
- `target.fx.type.id` ("1919251321")

Any feedback appreciated. Maybe there's a better wording for `type` but I can't come up with one.

**Comment by PucksEvilTwin** [comment]
I suppose my suggestion would be to drop ".type" all together.  I think they all make sense without that.  

If that messes with target.fx.name, maybe change our new placeholder to target.fx.displayname as it would be optimized for displays.

Sorry if that's unhelpful.  I'm terrible at coming up with syntax :)

**Comment by helgoboss** [comment]
Yeah, maybe the `.type` is a bit more qualifying than necessary. In this case, I would go for `target.fx.original_name` (instead of `target.fx.type.full_name`) and `target.fx.original_short_name` (instead of `target.fx.type.name`).
